pkg/core/interop_neo.go

```diff
@@ -605,7 +605,8 @@ func (ic *interopContext) contractMigrate(v *vm.VM) error {
 			v := ic.bc.GetTestVM(nil)
 			w := io.NewBufBinWriter()
 			emit.AppCallWithOperationAndArgs(w.BinWriter, hash, "decimals")
-			v.SetGasLimit(ic.bc.GetConfig().FreeGasLimit)
+			conf := ic.bc.GetConfig()
+			v.SetGasLimit(conf.GetFreeGas(0))
```

Comment:
Shouldn't we use current block index instead of `0`?

Reply:
Actually this was intentional, I wanted to avoid checking for current height and it's not a real call. At the same time, maybe it'd still be appropriate to be height-dependent here.